modules/mvt/src/lib/binary-vector-tile/vector-tile-feature.js

```diff
@@ -174,26 +178,72 @@ export default class VectorTileFeature {
   }
 }
 
-/**
- * In the original GeoJSON implementation this function
- * classifies the rings in a polygon into outlines & holes,
- * based on the vertex winding order. However the implementation
- * was broken and all it did was classify each ring as a separate
- * polygon outline. In order to maintain compatibility with the
- * original, we keep this behavior, but simplify the implementation
- * to an equivalent function:
- * [A, B, C] => [[A], [B], [C]]
- *
- * TODO in the future, the classification could be fixed following
- * https://github.com/mapbox/vector-tile-js, but this would require
- * confirming that the renderer of deck.gl is actually capable of
- * drawing polygons with holes in them.
- */
-function classifyRings(rings) {
-  return rings.map(r => [r]);
+// classifies an array of rings into polygons with outer rings and holes
+function classifyRings(geom) {
```

Comment:
Nit: Use proper JSDoc so we get intellisense in vscode and potentially type checking.

Is this algo MVT format specific or should it go into a common `@loaders.gl/math` package for processing geometries. Maybe add a comment to JSDoc header?

Reply:
Here I think the function should stay in place as it is specifically tied to format MVT tiles come in. In essence it prepares the rings for consumption by earcut. I'll improve the JSDoc as suggested